index.html

```diff
@@ -0,0 +1,254 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title> Gluten-free Goddess</title>
+  <link href='https://fonts.googleapis.com/css?family=Galada' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+<article> 
```

Comment:
Be aware of whitespace left in your files. Inevitably it will cause headaches in the form of merge conflicts. The best way to stem it is to either download the trailing spaces package for sublime to highlight your whitespace or add a line in your user config preferences to "trim_trailing_white_space_on_save": true.